web/templates/body.html

```diff
@@ -84,34 +89,70 @@ <h3 style="margin: 0; display: flex; align-items: center; justify-content: space
 </h2>
   <button id="Public-toggle" style="  
   display: flex; 
-        align-items: center;
-        justify-content: center;
+  align-items: center;
+  justify-content: center;
         width: 1.5rem;
         height: 1.5rem;
-flex-shrink: 0;
+        flex-shrink: 0;
 position: absolute;
-top: 0.75rem;
+top: 0.5rem;
 right: 1rem;
     border: none;
     background: transparent;
   ">
-    <svg id="public-chevron-path" width="24" height="24" fill="none" stroke="currentColor" stroke-width="1.5"
+
+
+    <svg width="24" height="24" fill="none" stroke="currentColor" stroke-width="1.5"
       viewBox="0 0 24 24" stroke-linecap="round" stroke-linejoin="round" xmlns="http://www.w3.org/2000/svg"
       style="transition: transform 0.3s ease;">
-      <path d="m6 6 6 6 6-6M6 12l6 6 6-6"/>
+      <path id = "public-chevron-path" d="m6 6 6 6 6-6M6 12l6 6 6-6"/>
     </svg>
   </button>
+
+
 </div>
 
 
   <div id="public-data-content">
-  <h3 style = "color: var(--theme-foreground, #020618);
-  font-family: var(--font-family-sans, Inter);
-  font-size: var(--font-size-base, 1rem);
-  font-style: normal;
-  font-weight: var(--font-weight-semibold, 600);
-  line-height: var(--font-leading-6, 1.5rem); /* 150% */
-  letter-spacing: var(--font-tracking-normal, 0rem);"
+
+    <div style=
+    
+    "margin-top: 2rem;
+    display: flex; 
+    align-items: flex-start; 
+    gap: 0.5rem; "
+    >
+
+    <button id="area-icon" style= "
```

Comment:
I realize this is WIP, but just a quick note that these styling details should probably live in the style.css longer term.